app/src/main/java/database/CrimeTypeConverters.kt

```diff
@@ -0,0 +1,30 @@
+package database
+
+import androidx.room.TypeConverter
+import java.util.*
```

Comment:
By default, it is a [bad practice](https://softwareengineering.stackexchange.com/questions/375901/is-importing-wildcards-a-bad-idea) to use _[wildcard imports](https://tharakamd-12.medium.com/is-it-bad-to-use-wildcard-imports-in-java-1b46a863b2be#:~:text=Wildcard%20imports%20tell%20java%20compiler,performance%20may%20lower%20a%20bit.)_. Check[ this link](https://stackoverflow.com/questions/67341957/how-do-i-disable-wildcard-imports-for-all-kotlin-files-in-android-studio) to see how you can prevent your IDE from doing it.

Reply:
Didn't know that. Thanks a lot. All wildcard imports have been deleted across all files.